First round of API v2 updates. (#2340)
XRPLF/xrpl-dev-portal: content/references/http-websocket-apis/public-api-methods/account-methods/noripple_check.md

|:---------------|:---------------------------|:-------------------------------|
| `account`      | String                     | A unique identifier for the account, most commonly the account's address. |
| `role`         | String                     | Whether the address refers to a `gateway` or `user`. Recommendations depend on the role of the account. Issuers must have Default Ripple enabled and must disable No Ripple on all trust lines. Users should have Default Ripple disabled, and should enable No Ripple on all trust lines. |

> | `transactions` | Boolean                    | [API v1][]: _(Optional)_ If `true`, include an array of suggested [transactions](transaction-formats.html), as JSON objects, that you can sign and submit to fix the problems. Defaults to false.<br>[API v2][]: Identical to v1, but also returns an `invalidParams` error if you use a non-boolean value. |

Reviewer: Embedding `<br>` tags in tables won't work under Redocly, so let's not do that.

While we're at it, we can update from "Defaults to ..." to "The default is ..." per our new(er than this page) style guide.

```suggestion
| `transactions` | Boolean                    | [API v1][]: _(Optional)_ If `true`, include an array of suggested [transactions](transaction-formats.html), as JSON objects, that you can sign and submit to fix the problems. The default is `false`. In [API v2][], providing a non-boolean value causes an `invalidParams` error. |
```
